koi/koi.py

```diff
@@ -36,11 +36,11 @@ def __init__(self):
         )
 
         self.steps = QSpinBox(self.input_widget)
-        self.steps.setRange(5, 150)
+        self.steps.setRange(5, 500)
```

Comment:
I believe some samplers may break with values larger steps, although admittedly I do not know where the cutoff is.

have you tested using 500 steps?